gap/Forms.gi

```diff
@@ -113,26 +129,34 @@ function(group, type)
             q := Size(F);
         fi;
     fi;
-    if type in ["O", "O+", "O-"] and IsEvenInt(q) then
-        ErrorNoReturn("ConjugateToGAPForm cannot deal with orthogonal groups in",
-                      " even characteristic yet");
-    fi;
     
     # get standard GAP form
     if type = "S" then
         gapForm := InvariantBilinearForm(Sp(d, q)).matrix;
     elif type = "U" then
         gapForm := InvariantSesquilinearForm(GU(d, q)).matrix;
     elif type = "O" then
-        gapForm := InvariantBilinearForm(GO(d, q)).matrix;
+        gapForm := InvariantBilinearForm(Omega(d, q)).matrix;
```

Comment:
Ad 1st comment. I would also favour option 2, but with one extra bit: We could simply agree on (and this would actually make much sense) always returning the group so that it preserves the same form as the "standard" version of the corresponding quasisimple group in GAP. Why is this senseful? Well, we actually want to construct subgroups _of these quasisimple groups_, so one would expect that they are least contained in the corresponding quasisimple group provided by GAP. (Using the form of the corresponding quasisimple group in GAP as a default is also a simple way of explaining what the `ConjugateToStandardForm` function does.) - Let's talk about this tomorrow though!

Ad 2nd comment. I guess this would actually not really work and for the following reason: It is also possible to use a group _without_ having none of the parameters `InvariantXForm` as an input for the function `ConjugateToStandardForm` - in this case, the invariant form is determined (at least for the unitary, symplectic and orthogonal odd char cases - yes, we should add orthogonal char 2 here) in this last line ``return ConjugateToSesquilinearForm(group, broadType, gapForm)``. In particular, the function `ConjugateToSesquilinearForm` needs to be told for which type of form to look so we cannot simply omit the argument `type` in these cases.